tests/unit/source_code/linters/test_pyspark.py

```diff
@@ -95,7 +96,7 @@ def test_spark_table_return_value_apply(migration_index):
     do_stuff_with_table(table)"""
     fixed_code = sqf.apply(old_code)
     # no transformations to apply, only lint messages
-    assert fixed_code == old_code
+    assert fixed_code.strip() == old_code.strip()
```

Comment:
I assume the problem here is that astroid introduces extra trailing newlines?

I think I'd prefer `.rstrip()` (which hints at the problem a little better) because changes to the start of the code indicate a problem.